packages/plugin-dev/src/node/hosted-plugins-manager.ts

```diff
@@ -78,19 +79,28 @@ export class HostedPluginsManagerImpl implements HostedPluginsManager {
         return this.runWatchScript(pluginRootPath);
     }
 
-    stopWatchCompilation(uri: string): Promise<void> {
-        const pluginPath = FileUri.fsPath(uri);
-
-        const watchProcess = this.watchCompilationRegistry.get(pluginPath);
-        if (!watchProcess) {
-            throw new Error('Watcher is not running in ' + pluginPath);
-        }
-
-        watchProcess.kill();
-        return Promise.resolve();
+  private killProcessTree(parentPid: number): void {
```

Comment:
something fishy with formatting, could you make sure to use Theia or VS Code to format this file

Reply:
i will reformat it. load a new commit soon

Reply:
done